index.html

```diff
@@ -2,9 +2,64 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
     <title>Tech It Easy Dashboard</title>
 </head>
 <body>
 
+<div class="container">
+    <div class=item id=rood>
```

Comment:
Ik zou "rood" niet als identifier willen gebruiken, maar naast dat is het ook niet in het Engels geformuleerd wat wél zou moeten 😉